task5_Ruben_Chiche/include/Distance.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include "Coordinate.h"
+
+class Distance {
```

Comment:
you don't need a class for doing the calculations, you can save a map with lambda expression, and do the calculations